src/grt/src/GlobalRouter.cpp

```diff
@@ -3294,12 +3294,12 @@ Net* GlobalRouter::addNet(odb::dbNet* db_net)
 void GlobalRouter::removeNet(odb::dbNet* db_net)
 {
   Net* net = db_net_map_[db_net];
-  if (net->isMergedNet()) {
+  if (net != nullptr && net->isMergedNet()) {
     fastroute_->mergeNet(db_net);
+    delete net;
   } else {
     fastroute_->removeNet(db_net);
```

Comment:
why is this here? seems unrelated to the PR

Reply:
When I added a new ODB callback function, this led to some GR fatals for some regressions or CI tests involving net removal.   This may be due to some order dependency between dbSta CB, GR CB and RSZ CB.  Adding RSZ CB may have changed the order.